### DISCUSSION TITLE
reexecute equivalent @urql/svelte 2

### DISCUSSION THREAD

**Discussion by riez** [urql-graphql/urql#2476] [Q&A] [answered]
Hi,

I see that the API are changes a lot, What is the equivalent command for `reexecute` on the new API? I'm trying it on the sveltekit right now using ssr.

**Comment by freb** [comment, marked as answer]
I'm no expert, but take a look at the example here:

https://github.com/FormidableLabs/urql/blob/main/examples/with-svelte/src/PokemonList.svelte#L5

Since the query `pokemons` is assigned using a reactive statement, it will update anytime one of its variables changes. Usually you should not need to re-run the query unless one of its inputs changes, at least if you're using group cache, as I understand it.

**Reply by JoviDeCroock** [reply]
Alternatively you can do something like

```js
const reexecute = () => {
  queryStore({ ...., requestPolicy: 'network-only' })
}
```

**Reply by riez** [reply]
That could work. Thanks.

cmiiw, on the prev. version we could subscribe on query that run on the server.

```
- on the context module
-- qrySmth on loadFunction

- on the client/script
-- use query(qrySmth) to subscribe
```

If i put store variable on the ssr component and update it by client. it means the data will be updated accordingly. am i correct? or I need to manually replacethe new data by doing the similar thing like @JoviDeCroock given example..?

**Reply by JoviDeCroock** [reply]
Not entirely sure what you mean here, if you mean you are leveraging the `ssrExchange` then as long as you restore the server-side data on the client-side creation off the exchange you will be benefiting from the server-side captured data.
